### ISSUE TITLE
[BUG] NX fails to detect .net projects dependencies if Ionic app presents in the repo as well

### ISSUE THREAD

**Issue by sukhanovr** [nx-dotnet/nx-dotnet#494]
**Describe the bug**
`nx-dotnet` utilizes ``${path}/**/*.*proj`` glob pattern to find .net project files within the workspace, so while building the graph for example `nx-dotnet` mistakenly assumes that `ionic`'s app with `project.pbxproj` file is a .net project as well. Which of course might cause multiple issues. In my case, nx fails to detect an explicit dependency between app and lib, while project reference is correctly set.

**To Reproduce**

1. `git clone https://github.com/sukhanovr/nx-dotnet-reproduction`
2. `npm i`
3. `npx nx@13 graph`

**Expected behavior**
There would be an explicit link between `dotnet-app` and `dotnet-lib`.

**Screenshots**
<img width="974" alt="image" src="https://user-images.githubusercontent.com/6266456/185098125-2f3140b3-f1f4-4111-a1d6-3cd29a208206.png">

```xml
  <ItemGroup>
    <ProjectReference Include="..\..\libs\dotnet-lib\zNxDotnetReproduction.DotnetLib.csproj" />
  </ItemGroup>
```

**Environment:**

- nx@13.10.6
- nx-dotnet/core@1.13.0

**Additional context**
While this is just one of the symptoms, there might be more issues with other `*proj` files in the workspace. I'm sure it's the root cause, problems happens here:
https://github.com/nx-dotnet/nx-dotnet/blob/master/packages/utils/src/lib/utility-functions/workspace.ts

```ts
 const xml: XmlDocument = new XmlDocument(
    readFileSync(absoluteNetProjectFilePath).toString(),
  );
  ```
  when absoluteNetProjectFilePath actually not present. I guess the solution would be to use exact C#, F#, VB patterns for any sort of lookup for .net projects. If I'm on the right path, pls let me know, I guess I can put a PR for this.

**Comment by AgentEnder** [comment]
Yeah, that makes sense. Definitely would appreciate a PR, should just be something like `${path}/**/*.@(fs|cs|vb)proj`

**Comment by sukhanovr** [comment]
Here is the pull request:
https://github.com/nx-dotnet/nx-dotnet/pull/495

**Comment by sukhanovr** [comment]
Here is the pull request:
https://github.com/nx-dotnet/nx-dotnet/pull/495

**Comment by AgentEnder** [comment]
Fixed in #495
